package.json

```diff
@@ -268,12 +271,14 @@
 		"jsdom": "22.1.0"
 	},
 	"scripts": {
-		"build": "npm run build:packages && wp-scripts build",
+		"build": "npm run build:packages && concurrently 'npm run build:plugins' 'wp-scripts build'",
```

Comment:
The question to be asked is whether we want to build plugins when we're working on Gutenberg. Building plugins for now is probably very fast but I expect that as we increase the number of canonical blocks, it might become a burden and developers would want to focus on one thing in particular: Gutenberg, a specific block...  An option to build everything, would be handy too but I wonder if it should be the default or not.

Reply:
My personal expectation is that the primary build command will build the entire project, and that I shouldn't need to run any other commands for a complete build. That said, having a `build:gutenberg` or similar that only builds the main Gutenberg plugin (and packages) seems like a good idea.